ceph/ceph.py

```diff
@@ -607,9 +607,12 @@ def get_osd_metadata(self, osd_id, client=None):
                 return metadata
         return None
 
-    def osd_check(self, client):
+    def osd_check(self, rhbuild, client):
+        cmd = "ceph -s -f json"
+        if rhbuild.startswith("5"):
```

Comment:
Continuing from comment 1, we could do `if rhbuild and rhbuild.startswith("5"):`

Reply:
done.